### PR TITLE
RK-20649 - Dt desktop dt reskin remove bugsnag

### PR DESCRIPTION
**Pull request by NoaAylonDyn**
@ron-rookout

### CODE REVIEW

**Comment by dan505512** [comment]
@NoaAylonDyn Do you feel you're ready to review this for @ron-rookout ? If so, please review and fix whatever you think. Go over it together and I'll mark the approval when you both tell me it's ready

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "explorook",
-  "version": "1.16.5",
+  "version": "1.17.0",
```
Comment (NoaAylonDyn) [inline package.json:3]: align to current package.json version

File: src/exceptionManager.ts [hunk 2]
```diff
@@ -1,78 +1,18 @@
...
-
-export const leaveBreadcrumb = (name: string, metaData?: any, type?: string, timestamp?: string) => {
-  exceptionManagerInstance?.leaveBreadcrumb(name, metaData, type, timestamp);
-};
-
 export class Logger {
 
   public info(message?: any) {
-    exceptionManagerInstance?.leaveBreadcrumb("log", { message }, "info");
     console.info(message);
```
Comment (NoaAylonDyn) [inline src/exceptionManager.ts:13]: why console?

File: src/exceptionManager.ts [hunk 3]
```diff
@@ -1,78 +1,18 @@
...
   }
   public warn(message?: any) {
-    exceptionManagerInstance?.leaveBreadcrumb("log", { message }, "warn");
     console.warn(message);
```
Comment (NoaAylonDyn) [inline src/exceptionManager.ts:16]: also here, we need to write logs to a different place, even to a local file or logger..

File: src/webapp/src/App.css [hunk 4]
```diff
@@ -1,30 +1,24 @@
...
-  color: #6E7A89;
+  color: #AEAFC980;
 }
 
 body {
   height: 100%;
   -ms-overflow-style: scrollbar;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
```
Comment (NoaAylonDyn) [inline src/webapp/src/App.css:21]: think we need to use DT font

**Comment by NoaAylonDyn** [comment]
@dan505512 not see anything risky at this point, just need to check after creating a release that all packages are supported on the new machine.

File: src/exceptionManager.ts (hunk 2 above)
Comment (dan505512) [reply src/exceptionManager.ts:13]: Can you check if the logfile still gets created? I don't think it was bugsnag managing it for us

**Comment by dan505512** [comment]
@NoaAylonDyn please resolve conflicts and I'll approve

**Comment by NoaAylonDyn** [comment]
@dan505512  can u approve?

**Review by dan505512 — APPROVED** [review]

File: src/deeplinks.ts [hunk 5]
```diff
@@ -20,5 +20,5 @@ export const initDeeplinks = (app: Electron.App) => {
 
 
 export const deeplinkHandler = () => {
-   dialog.showMessageBoxSync({title: "Rookout Desktop App", message: "Rookout Desktop App is now running"});
+   dialog.showMessageBoxSync({title: "Dynatrace Desktop App", message: "Dynatrace Desktop App is now running"});
```
Comment (Urook) [inline src/deeplinks.ts:23]: @NoaAylonDyn @ron-rookout Shouldn't the protocol be changed too? It is still "rookout"

Comment (Urook) [reply src/deeplinks.ts:23]: > @NoaAylonDyn @ron-rookout Shouldn't the protocol be changed too? It is still "rookout"

@dan505512 you too i guess lol

Comment (Urook) [reply src/deeplinks.ts:23]: Also under "protocols" in package.json

**Pull request merged**